scripts/copyExampleProjectFiles.js

```diff
@@ -0,0 +1,153 @@
+const path = require('path')
+const fs = require('fs')
+
+const TEST_PROJECT_DIR_NAME = 'test_project'
+const EXAMPLE_PROJECT_DIR_NAME = 'example'
```

Comment:
tbh you can create `EXAMPLE_PROJECT_ROOT = '../example'` cuz `EXAMPLE_PROJECT_DIR_NAME` just adds work though it is always the same pattern `../EXAMPLE_PROJECT_DIR_NAME`